hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageDispatcherService.java

```diff
@@ -500,4 +505,58 @@ private List<SoftwareModuleMetadata> getSoftwareModuleMetadata(final SoftwareMod
                 PageRequest.of(0, RepositoryConstants.MAX_META_DATA_COUNT), module.getId()).getContent();
     }
 
+    private void sendUpdateMessageToTargets(final Map<String, ActionProperties> actions, final List<Target> targets,
+                                            final Map<SoftwareModule, List<SoftwareModuleMetadata>> modules) {
```

Comment:
Mark arguments and local variables final

Reply:
all arguments are final